README.md

```diff
@@ -12,6 +12,7 @@ Once they're ready, we'll officially push the components as version 1.0.0.
 <!--- keep these alphabetical --->
 | Package | Status | Description |
 | --- | --- | --- |
+| [@labkey/build](packages/build/README.md) | | UI themes for LabKey Server.
```

Comment:
I think that description is not what you intended.

Reply:
Argh....I could have sworn I changed that locally after copy/pasting. Done.